src/app/_components/FeaturedEcosystemProjects.tsx

```diff
@@ -1,34 +1,44 @@
-import { Badge } from '@/components/Badge'
-import { Heading } from '@/components/Heading'
-import { TextLink } from '@/components/TextLink'
+import { MagnifyingGlass } from '@phosphor-icons/react/dist/ssr'
+
+import { Card } from '@/components/Card'
+import { CardLayout } from '@/components/CardLayout'
 
 import { getEcosystemProjectsData } from '@/utils/getEcosystemProjectData'
+import { shuffleArray } from '@/utils/shuffleArray'
 
 import { PATHS } from '@/constants/paths'
 
 const ecosystemProjects = getEcosystemProjectsData()
-
-const featuredEcosystemProjects = ecosystemProjects.filter(
-  (project) => project.featured,
-)
+const FEATURED_PROJETS_LIMIT = 6
 
 export function FeaturedEcosystemProjects() {
-  if (featuredEcosystemProjects.length === 0) {
+  const featuredEcosystemProjects = shuffleArray(ecosystemProjects).slice(
+    0,
+    FEATURED_PROJETS_LIMIT,
+  )
+
+  if (featuredEcosystemProjects.length < FEATURED_PROJETS_LIMIT) {
     return <p>No featured projects available.</p>
   }
 
   return (
-    <ul className="list-none gap-8 sm:grid sm:grid-cols-2 sm:gap-6 md:grid-cols-3 lg:grid-cols-4">
-      {featuredEcosystemProjects.map(({ slug, title, description }) => (
-        <li key={slug} className="ml-0">
-          <Badge>Featured Project</Badge>
-          <Heading tag="h3" variant="lg">
-            {title}
-          </Heading>
-          <p>{description}</p>
-          <TextLink href={`${PATHS.ECOSYSTEM.path}/${slug}`}>See More</TextLink>
-        </li>
+    <CardLayout>
```

Comment:
gap is 24px in the designs.

Reply:
When you notice inconsistencies like these - It's probably an oversight on design sides. And it's a good idea to check [>>](https://filecoinfoundation.slack.com/archives/C06MTEQ3SP6/p1714658929642879)